build_projects/shared-build-targets-utils/VersionRepoUpdater.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.DotNet.Cli.Build
     public class VersionRepoUpdater
     {
         private static Regex s_nugetFileRegex = new Regex("^(.*?)\\.(([0-9]+\\.)?[0-9]+\\.[0-9]+(-([A-z0-9-]+))?)\\.nupkg$");
-
+        private static string s_baseGitHubPath = $"/build-info/dotnet/cli";
```

Comment:
Shipping it via core-setup would be possible and straightforward but given that, that repo is soon moving to MSBuild in the short-term it didn't seem worth the investment to make it happen. 

If you feel strongly about it, I wouldn't be opposed to a PR enabling this.